app/models/changeset/check_suite.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+class Changeset::CheckSuite
+  attr_reader :repo, :data
+
+  def initialize(repo, data)
+    @repo = repo
+    @data = data
+  end
+
+  def self.changeset_from_webhook(project, payload)
+    new(project.repository_path, payload)
+  end
+
+  def self.valid_webhook?(payload)
+    payload['action'] == 'completed' && payload['check_suite']['status'] == 'completed' && payload['check_suite']['conclusion'] == 'success'
```

Comment:
does it need to check all 3 ?
I'd imagine a `completed` hook has status 'completed' too

Reply:
The use case is we want to send a webhook to inform Samson once all the jobs within a Github workflow have completed. This is per [Github official recommendation](https://github.community/t/official-webhook-action/116972), but currently no such action exists.